CMakeLists.txt

```diff
@@ -1744,7 +1744,7 @@ set(CONFIG_DIR "${CMAKE_INSTALL_PREFIX}/etc/netdata")
 set(LIBCONFIG_DIR "${CMAKE_INSTALL_PREFIX}/usr/lib/netdata/conf.d")
 set(LOG_DIR "${CMAKE_INSTALL_PREFIX}/var/log/netdata")
 set(PLUGINS_DIR "${CMAKE_INSTALL_PREFIX}/usr/libexec/netdata/plugins.d")
-set(WEB_DIR "${CMAKE_INSTALL_PREFIX}/usr/share/netdata/web")
+set(WEB_DIR "${CMAKE_INSTALL_PREFIX}/var/lib/netdata/www")
```

Comment:
/var/lib/netdata/www is Debian native only

Reply:
so you mean this will affect also other distros, and we don't want this?

Reply:
we also do this 
https://github.com/netdata/netdata/blob/master/contrib/debian/rules#L204-L208
for debian, let me check side to side the packages produced for both rpm and deb

Reply:
debian native was the only place where we set web dir (see `--with-webdir`)

https://github.com/netdata/netdata/blob/5d998b946fedddbde106309d57b64ec96f596332/contrib/debian/rules#L43